src/components/app/AppViewRouter.js

```diff
@@ -89,18 +89,15 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
             message = 'AppViewRouter--error-from-localhost-url-safari';
           } else {
             console.error(view.error);
+            message = view.error.message;
             additionalMessage = (
-              <>
-                <p>{view.error.toString()}</p>
-                <p>The full stack has been written to the Web Console.</p>
-              </>
+              <p>The full stack has been written to the Web Console.</p>
             );
           }
         }
 
         return (
           <Localized
-            id={message}
```

Comment:
fyi: not passing an `id` generates a console warn. We could technically remove the `<Localized />` component, but I feel that keeping the diff as small as possible will make rebases less noisy.

<img width="1279" alt="CleanShot 2022-12-29 at 14 33 38@2x" src="https://user-images.githubusercontent.com/32957606/210017434-60e5c013-f0a7-45e0-b348-ba5e84c88ed9.png">